src/handlers/unixfs-dir.js

```diff
@@ -61,7 +61,7 @@ export async function handleUnixfsDir (request, env, ctx) {
 
   const headers = {
     'Content-Type': 'text/html',
-    Etag: `"DirIndex-dagula@0.0.0_CID-${entry.cid}"`
+    Etag: `"DirIndex-gateway-lib@2.0.2_CID-${entry.cid}"`
```

Comment:
Invalidates existing directories (since we've changed the response) and also more accurately describes what is generating the response!